tgui/packages/tgui/interfaces/_FaxMachine.js

```diff
@@ -0,0 +1,286 @@
+                      A copy of the sent paper will print, too - for record-keeping."}
+                      onClick={() => act('send_stored_paper', {
+                        destination_machine: destination,
+                      })} />
+                  </Stack.Item>
+                  <Stack.Item grow>
+                    <Dropdown
+                      width="100%"
+                      height="100%"
+                      selected={selectedDestination}
+                      options={destination_options}
+                      onSelected={dest => { setDestination(dest); }} />
+                  </Stack.Item>
+                </Stack>
+              )}
+              {tab === 2 && received_paper && (
+                <Stack>
+                  <Stack.Item>
+                    <Button
+                      disabled={!received_paper}
+                      content="Print received Fax"
+                      tooltip="Print the last received fax."
+                      onClick={() => act('print_received_paper')} />
+                  </Stack.Item>
+                </Stack>
+              )}
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <Section
+          title="Paperwork"
+          buttons={(
+            <Button.Checkbox
+              checked={can_receive}
+              content="Toggle Incoming Paperwork"
+              tooltip={(can_receive ? "Disable" : "Enable")
+                + " the ability for this fax machine \
+                to receive paperwork every five minutes."}
+              onClick={() => act('toggle_recieving')}
+            />
+          )}>
+          <Stack vertical grow>
+            <Stack.Item height={2}>
+              {received_paperwork && received_paperwork.length > 0 ? (
+                <Tabs>
+                  {received_paperwork.map(paper => (
+                    <Tabs.Tab
+                      width="12.5%"
+                      key={paper}
+                      textAlign="center"
+                      selected={paper.ref === selectedPaperTab}
+                      onClick={() => setSelectedPaper(paper.ref)}>
+                      Paper {paper.num}
+                    </Tabs.Tab>
+                  ))}
+                </Tabs>
+              ) : (
+                "No stored paperwork to process."
+              )}
+            </Stack.Item>
+            <Stack.Item height={8}>
+              {selectedPaper ? (
+                <BlockQuote fontSize="14px">
+                  {selectedPaper.contents}
+                </BlockQuote>
+              ) : (
+                received_paperwork && received_paperwork.length > 0
+                && ("No paper selected.")
+              )}
+            </Stack.Item>
+            <Divider />
+            <Stack.Item height={3} mb={1}>
+              {!!selectedPaper && (
+                <Stack vertical align="center" >
+                  <Stack.Item grow>
```

Comment:
```suggestion
                  <Stack.Item>
```
will fix this 
![image](https://user-images.githubusercontent.com/71735193/167250496-f62810c1-46a1-4795-a11d-a0b5aee02a17.png)
after:
![image](https://user-images.githubusercontent.com/71735193/167250509-2c5e60d7-49f8-4442-8fbb-271b997aa301.png)